src/func.d

```diff
@@ -4486,7 +4486,7 @@ public:
         return this;
     }
 
-    override const(char)* kind()
+    override const(char)* kind() const
     {
         // GCC requires the (char*) casts
         return (tok != TOKfunction) ? cast(char*)"delegate" : cast(char*)"function";
```

Comment:
Can you also fix.... ;-)

About your comments on 5k PRs, I've fallen into the habit of reviewing individual commits.  I think you should too.